schema/deploy/shipping/views.sql

```diff
@@ -3316,3 +3318,83 @@ grant select
   to "uw-priority-queue-processor";
 
 commit;
+
+
+create or replace view shipping.linelist_data_for_wa_doh_v1 as (
+    select distinct on (sample_id)
+          -- The naming scheme of these columns is an artefact of how we
+          -- originally submitted data to WaDoH from REDCap reports. Now they
+          -- expect columns structured and named in a specific way, and we
+          -- cannot modify those with our current linelist format. We can notify
+          -- WaDoH of our intent to change our submitted data format (e.g.
+          -- custom format or ELFF), but they'll need sufficient advanced
+          -- notice. At that point, we can update this view with different
+          -- column names.
+          --
+          -- kfay, 5 January 2021
+          sample_id,
+          sampleid.barcode as sample_barcode,
+          collectionid.barcode as collection_barcode,
+          sample.details ->> 'clia_barcode' as scan_id,
+          case
+              when present then 'positive'
+              when present = 'f' then 'negative'
+              else 'inconclusive'
+          end as test_result,
+          hcov19_result_release_date as date_tested,
+          best_available_encounter_date as enrollment_date,
+          collected as collection_date,
+          best_available_site as site_name,
+          best_available_site_type as site_context,
+          sex as sex_new,
+          case
+            when hispanic_or_latino then 'yes'
+            when hispanic_or_latino = 'f' then 'no'
```

Comment:
Same comment as above. Either use 't' and 'f' or when hispanic_or_latino or when not hispanic_or_latino.